### PR TITLE
Add E2E test for TableOfContentsPlugin

### PR DESCRIPTION
**Pull request by karam-qaoud**

### CODE REVIEW

File: packages/lexical-playground/__tests__/utils/index.mjs [hunk 1]
```diff
@@ -385,6 +389,16 @@ export async function getHTML(page, selector = 'div[contenteditable="true"]') {
   return element.innerHTML();
 }
 
+export async function getElement(
```
Comment (thegreatercurve) [inline packages/lexical-playground/__tests__/utils/index.mjs:392]: @karam-qaoud Can you not just use `getEditorElement` to get your element? The logic seems basically the same

**Review by thegreatercurve — COMMENTED** [review]
Might need to delete the `packages/lexical-playground/src/plugins/.TableOfContentsPlugin.tsx.swp` file that keeps being committed, or add it to the .gitignore.

File: packages/lexical-playground/__tests__/utils/index.mjs (hunk 1 above)
Comment (karam-qaoud) [reply packages/lexical-playground/__tests__/utils/index.mjs:392]: getEditorElement gets you the editor itself, but this method gets you any element within the editor/page.

Comment (thegreatercurve) [reply packages/lexical-playground/__tests__/utils/index.mjs:392]: Can we refactor `getEditorElement` to use `getElement` then? Keeps things more DRY.

**Review by thegreatercurve — APPROVED** [review]

**Review by zurfyx — APPROVED** [review]

**Comment by thegreatercurve** [comment]
@karam-qaoud Does this fix the page freezing issue

**Comment by karam-qaoud** [comment]
@thegreatercurve Yes, `packages/lexical-playground/src/plugins/TableOfContentsPlugin/index.tsx` has the fix.

**Pull request merged**